aatp/peerbook_webrtc/admin.spec.ts

```diff
@@ -350,4 +350,28 @@ test.describe('peerbook administration', ()  => {
         const twr = await getTWRBuffer(page)
         expect(twr).toMatch(/Logged in/)
     })
+    test('test the support command', async ({request}) => {
+        await sleep(100)
+        await page.keyboard.type('support')
+        await page.keyboard.press("Enter")
+        await sleep(100)
+        const twr = await getTWRBuffer(page)
+        expect(twr).toMatch(/address:$/)
+        await page.keyboard.type('test@gmail.com')
+        await page.keyboard.press("Enter")
+        await sleep(100)
+        await page.keyboard.press("ArrowDown")
+        await page.keyboard.press("Enter")
```

Comment:
Before the enter, please add a log message so you can later test the log is also sent. Use `page.evaluate(...)` to call terminal7.log directly.

Reply:
i did something i hope its good